tests/zenko_tests/node_tests/backbeat/ReplicationUtility.js

```diff
@@ -261,15 +301,17 @@ class ReplicationUtility {
                     },
                     UploadId: uploadId,
                 };
-                return this.s3.completeMultipartUpload(params, next);
+                return this.s3.send(new CompleteMultipartUploadCommand(params))
+                    .then(data => next(null, data))
+                    .catch(next);
             },
         ], (err, data) => {
             if (err) {
-                return this.s3.abortMultipartUpload({
+                return this.s3.send(new AbortMultipartUploadCommand({
                     Bucket: bucketName,
                     Key: objectName,
                     UploadId: uploadId,
-                }, () => cb(err));
+                })).then(() => cb(err)).catch(() => cb(err));
```

Comment:
```suggestion
                })).finally(() => cb(err));
```

If you have an error in the Abort, it'll be a silent error